astrosat/views.py

```diff
@@ -1,4 +1,7 @@
+import logging
+import json
 from itertools import filterfalse
+from datetime import datetime
 
```

Comment:
`datetime` is not used in this module

Reply:
Removed the code but forgot about this, will remove